### PR TITLE
Chapter 3 Challenge 2 (Coop-lose)

### PR DESCRIPTION
**Pull request by vinayaksh42**
Fixes: #505 

[Preview](https://saving-satoshi-git-fork-vinayaksh42-issue505-savingsatoshi.vercel.app/en/chapters/chapter-3/coop-1?dev=true)

To-Do:
- Removal of sample coop challenge page.
- Fix the order of challenges (can be done in a separate issue).

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
Tested. Looks like the order of challenges is a bit wonky. It currently goes from challenge 2 (Coop-lose) to challenge 1 (solo vs bitrey lose) to challenge 2 (Coop-lose).

**Comment by ecurrencyhodler** [comment]
Also this comment is not a blocker but @GBKS do we need a different profile picture for bitrey?

Screenshot 2023-07-22 at 8 33 29 AM

**Comment by vinayaksh42** [comment]
> Tested. Looks like the order of challenges is a bit wonky. It currently goes from challenge 2 (Coop-lose) to challenge 1 (solo vs bitrey lose) to challenge 2 (Coop-lose).

The order will be fixed once either my PR is merged or @benalleng's PR is merged. The Bitrey picture is also different, check [this](https://saving-satoshi-dxkcu95nz-savingsatoshi.vercel.app/en/chapters/chapter-3/coop-1?dev=true). The solo and coop challenge you see in the preview of this PR will be removed.

**Comment by ecurrencyhodler** [comment]
awesome! Okay then it's a tACK for me.

File: content/lessons/chapter-3/coop-1.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+export default function Coop1({ lang }) {
+  const t = useTranslations(lang)
+
+  const saveAndProceed = useSaveAndProceed()
+  const isSmallScreen = useMediaQuery({ width: 767 })
+  const [playersFound, setPlayersFound] = useState(false)
+  const [players, setPlayers] = useState([
+    {
+      username: 'You',
+      avatar: '/assets/avatars/1.png',
```
Comment (afterburn) [inline content/lessons/chapter-3/coop-1.tsx:25]: We should probably try to take the avatar of the currently logged in user (if it exists) and fallback to this one.

**Review by afterburn — CHANGES_REQUESTED** [review]
Requested small change

**Review by afterburn — APPROVED** [review]

**Pull request merged**